.github/workflows/build-maven.yml

```diff
@@ -8,11 +8,6 @@ on:
 jobs:
   build:
     runs-on: ubuntu-latest
-    strategy:
-      matrix:
-        java: [ '8', '11' ]
```

Comment:
:wrench: Let's change the matrix to compile with Java 11 and 17. 

So, we can ensure that FEEL is working with the Java versions of C7 and C8. :rocket:

Reply:
Super. :+1: